apps/rush-lib/src/logic/PublishGit.ts

```diff
@@ -44,13 +44,13 @@ export class PublishGit {
       workingDirectory ? workingDirectory : process.cwd());
   }
 
-  public addTag(shouldExecute: boolean, packageName: string, packageVersion: string): void {
+  public addTag(shouldExecute: boolean, packageName: string, packageVersion: string, commitId: string | undefined): void {
```

Comment:
```suggestion
  public addTag(shouldExecute: boolean, packageName: string, packageVersion: string, commitId?: string): void {
```

Reply:
I'm open to debate, but I'm a bigger fan of requiring `undefined` vs omitting the parameter here... IMO, it should be intentional that an undefined commitId is passed to cut down on cases where someone would tag without a commit id. 

It's fine to pass undefined, but I think the likely case for omitting a commit id is programmer error (ie forgot to pass the cli value through).